scripts/02227-KimayaUrdhwareshe

```diff
@@ -0,0 +1,6 @@
+Name = "Kimaya Urdhwareshe"
+ID = "HNG-02227"
+Language = "Python"
+
+
+print("Hello World, this is {0} with HNGi7 ID {1} using {2} for stage 2 task.".format(Name, ID, Language))
```

Comment:
Please add your email address to this out. Here's a sample format `Hello World, this is Ciroma Adekunle with HNGi7 ID HNG-1010 using Javascript for stage 2 task abc@example.com`